deploy/02_mint_t.ts

```diff
@@ -0,0 +1,37 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+import { BigNumber } from "ethers"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+  const { execute, read, log } = deployments
+
+  const T = await deployments.get("T")
+
+  // We're minting 10B T, which is the value of the T supply on the production
+  // environment.
+
+  await execute(
+    "T",
+    { from: deployer },
+    "mint",
+    deployer,
+    BigNumber.from(10).pow(28)
+  )
+
+  const tTotalSupply = await read("T", "totalSupply")
+
+  log("minted", tTotalSupply.toString(), "T")
+
+  if (hre.network.tags.tenderly) {
+    await hre.tenderly.verify({
+      name: "T",
+      address: T.address,
+    })
+  }
```

Comment:
We already verified the contract in the `01_deploy_t.ts` script.
```suggestion
```

Reply:
Removed, see 5c37de311011d10d547ccc85c2cbb47e25b8f304.